EIPS/eip-draft-evm-profile-for-type3-zkrollups.md

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
```suggestion
---
eip: 7530
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.